Index: src/app/signup/UserDTO.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>class UserDTO {\r\n    name!: string;\r\n    email!: string;\r\n    password!: string;\r\n}
===================================================================
diff --git a/src/app/signup/UserDTO.ts b/src/app/signup/UserDTO.ts
--- a/src/app/signup/UserDTO.ts	
+++ b/src/app/signup/UserDTO.ts	
@@ -1,4 +1,4 @@
-class UserDTO {
+export default  class UserDTO {
     name!: string;
     email!: string;
     password!: string;
